subprojects/common/time/src/main/kotlin/com/avito/time/DefaultTimeProvider.kt

```diff
@@ -50,9 +40,7 @@ class DefaultTimeProvider(loggerFactory: LoggerFactory) : TimeProvider {
             cal1.get(Calendar.DAY_OF_YEAR) == cal2.get(Calendar.DAY_OF_YEAR)
     }
 
-    override fun formatter(pattern: String): DateFormat {
-        return SimpleDateFormat(pattern, locale).apply {
-            timeZone = this@DefaultTimeProvider.timeZone
-        }
+    override fun utcFormatter(pattern: String): DateFormat = SimpleDateFormat(pattern, locale).apply {
+        timeZone = TimeZone.getTimeZone("UTC")
     }
```

Comment:
I propose to extract this function from TimeProvider.
I don't see reasons to have it TimeProvider's interface.

We can create SimpleDateFormat anywhere, it's a simple class:

 ```kotlin
val utcFormat: DateFormat = SimpleDateFormat(pattern, Locale.getDefault()).apply {
        timeZone = TimeZone.getTimeZone("UTC")
    }
 ```

Reply:
fixed